test/common/dashboards_data.go

```diff
@@ -78,15 +78,20 @@ var (
 	}
 )
 
+func TestDashboardsDataObservability(t TestingTB, ctx *TestingContext){
+	execToPodCommand := "wget -qO - "
+	testDashboardsData(t, ctx, ObservabilityProductNamespace, execToPodCommand)
+}
+
 // TestDashboardsData verifies that all dashboards are installed and all the graphs are filled with data
-func TestDashboardsData(t TestingTB, ctx *TestingContext) {
+func testDashboardsData(t TestingTB, ctx *TestingContext, monitoringNamespace string, execToPodCommand string) {
```

Comment:
same question about monitoring namespace as in above file

Reply:
Done in new PR: https://github.com/integr8ly/integreatly-operator/pull/2188